GitHubUserSearch/GitHubUserSearch/Model/Model.swift

```diff
@@ -135,29 +137,39 @@ class Model {
             performOnMainThreadAsync {
                completion?()
             }
+         } else if let anError = error {
+            print(anError)
+            performOnMainThreadAsync {
+               completion?()
+            }
          }
       }
    }
 }
 
-//MARK: -
+// MARK: -
 struct UserSearchResultResponse:Decodable {
-   var total_count:Int?
-   let incomplete_results:Bool?
+   var totalCount:Int?
+   let incompleteResults:Bool?
    var items:[UserData]?
 
    static func createEmpty() -> UserSearchResultResponse {
-      return UserSearchResultResponse(total_count: nil, incomplete_results: false, items: nil)
+      return UserSearchResultResponse(totalCount: nil, incompleteResults: false, items: nil)
    }
    
    mutating func appendUsers(_ items:[UserData]) {
-      if let _ = self.items {
+      if self.items != nil {
          self.items?.append(contentsOf:items)
-      }
-      else {
+      } else {
          self.items = items
       }
    }
+   
+   private enum CodingKeys: String, CodingKey {
+      case totalCount = "total_count"
+      case incompleteResults = "incomplete_results"
+      case items
+   }
 }
 
 struct UserData:Decodable {
```

Comment:
Could you please explain, why you decide to placed `UserData`, `UserModel`, `UserSearchResultResponse`, `RepositoryData`, `RequestError` at one file?

Reply:
the data structs are pretty small, easy to see them in a file while developing.
they can be put outside to a separate file each.